postgres/check.py

```diff
@@ -716,6 +716,8 @@ def _get_custom_metrics(self, custom_metrics, key):
     def check(self, instance):
         host = instance.get('host', '')
         port = instance.get('port', '')
+        if port != '':
+            port = int(port)
```

Comment:
Are we doing this to work around some issue with chef or puppet recipes? The port should really be specified as an integer in the YAML... 🤔 

That said, YAML parsers are imperfect, so there's no harm done here.